Sources/ProjectSpec/Project.swift

```diff
@@ -193,7 +193,19 @@ extension Project {
         attributes = jsonDictionary.json(atKeyPath: "attributes") ?? [:]
         include = jsonDictionary.json(atKeyPath: "include") ?? []
         if jsonDictionary["packages"] != nil {
-            packages = try jsonDictionary.json(atKeyPath: "packages", invalidItemBehaviour: .fail)
+            packages = try jsonDictionary.json(atKeyPath: "packages", invalidItemBehaviour: .custom({ error in
```

Comment:
Unless I'm missing something, the fix could live directly in `SwiftPackage.VersionRequirement(jsonDictionary: JSONDictionary)`

Reply:
Hello @yonaskolb 
It's possible since I'm not very familiar with the project)